web/src/lib/components/album-page/album-title.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import { updateAlbumInfo } from '@immich/sdk';
+  import { NotificationType, notificationController } from '../shared-components/notification/notification';
+  import { handleError } from '$lib/utils/handle-error';
+
+  export let id: string;
+  export let albumName: string;
+  export let isOwned: boolean;
+
+  let titleInput: HTMLInputElement;
+  $: newAlbumName = albumName;
+
+  const handleUpdateName = async () => {
+    if (newAlbumName === albumName) {
+      return;
+    }
+
+    try {
+      await updateAlbumInfo({
+        id,
+        updateAlbumDto: {
+          albumName: newAlbumName,
+        },
+      });
+      albumName = newAlbumName;
+      notificationController.show({ type: NotificationType.Info, message: 'New album name has been saved' });
+    } catch (error) {
+      handleError(error, 'Unable to update album name');
+    }
+  };
+</script>
+
+<input
+  on:keydown={(e) => e.key === 'Enter' && titleInput.blur()}
```

Comment:
I think you can also get the titleInput with something like `e.target`, right? That would remove the extra variable